lcm-cmake/cpack.cmake

```diff
@@ -6,31 +6,28 @@ set(CPACK_GENERATOR TGZ STGZ)
 
 # Detect OS type, OS variant and target architecture
 if(UNIX)
-    if(APPLE)
-        set(OS_TYPE macos)
-    else()
-        set(OS_TYPE linux)
-
-        # Determine distribution
-        execute_process(COMMAND lsb_release -si
-            OUTPUT_VARIABLE LINUX_DISTRO
-            OUTPUT_STRIP_TRAILING_WHITESPACE
-        )
-
-        # Add CPack generator according to detected distribution
-        if(LINUX_DISTRO STREQUAL "Debian" OR LINUX_DISTRO STREQUAL "Ubuntu" OR LINUX_DISTRO STREQUAL "LinuxMint")
-            list(APPEND CPACK_GENERATOR DEB)
-        endif()
-    endif()
-
-    # Determine architecture
+  if(APPLE)
+    set(OS_TYPE macos)
+  else()
+    set(OS_TYPE linux)
+  endif()
+  # Determine architecture
+  find_program(DPKG_EXECUTABLE dpkg)
```

Comment:
Do we really want to try for .deb even on macOS? (Honest question; maybe we do? If not, suggest moving the `find_program` into the `else()` and make the test `DEFINED DPKG_EXECUTABLE AND DPKG_EXECUTABLE`.)

Reply:
Your solution seems reasonable. I left it for MacOS because it allowed to keep only a simple 'if' statement, but your solution is still simple.